File: core/server/project/commonBase/src/main/java/com/home/commonBase/trigger/TriggerInstance.java
package com.home.commonBase.trigger;

import com.home.shine.data.trigger.TriggerConfigData;
import com.home.shine.support.collection.SSet;
import com.home.shine.support.pool.IPoolObject;

/** trigger实例 */
public class TriggerInstance implements IPoolObject
{
	/** 配置 */
	public TriggerConfigData config;
	/** 间隔组 */
	public SSet<TriggerIntervalData> intervalDic=new SSet<>(TriggerIntervalData[]::new);
	
	@Override
	public void clear()
	{
		config=null;
	}
}

File: core/client/game/src/commonGame/trigger/TriggerInstance.cs
using System;
using ShineEngine;

/// <summary>
/// trigger实例
/// </summary>
public class TriggerInstance:IPoolObject
{
	/** 配置 */
	public TriggerConfigData config;
	/** 间隔组 */
	public SSet<TriggerIntervalData> intervalDic=new SSet<TriggerIntervalData>();

	public void clear()
	{
		config=null;
	}
}